presto-docs/src/main/sphinx/admin/session-properties-manager.rst

```diff
@@ -0,0 +1,21 @@
+==========================
+Session Properties Manager
+==========================
+
+A session properties manager plugin provides a way to dynamically override default
+session properties. It supports overriding default session properties for each
+session based on the ``SessionConfigurationContext``, which includes ``user``,
+``source``, and ``clientTags``. The returned properties override the default
+values. However, session properties set by users are always respected, and take
+precedence over these values.
+
+Implementation
```

Comment:
I don’t think this is useful for documentation. It basically just repeats the Javadoc and is the type of thing that will become out of date as the code changes.

Reply:
In that case I don't really see it necessary to have documentation for this feature. We don't have a open source plugin implemented, so there's not much of "how to configure it" to document.